lighthouse-core/scripts/generate-bundlephobia-database.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+'use strict';
+
+/* eslint-disable no-console */
+
+// TODO(saavan) These type definitions are incorrect and need to be fixed.
+/** @typedef {{gzip: number, name: string, version: string | number, repository: string}} BundlePhobiaLibrary */
+
+const fs = require('fs');
+const getPackageVersionList = require('bundle-phobia-cli').fetchPackageStats.getPackageVersionList;
+const fetchPackageStats = require('bundle-phobia-cli').fetchPackageStats.fetchPackageStats;
+
+const databasePath = '../audits/byte-efficiency/bundlephobia-database.json';
```

Comment:
```suggestion
const fs = require('fs');
const path = require('path');
const getPackageVersionList = require('bundle-phobia-cli').fetchPackageStats.getPackageVersionList;
const fetchPackageStats = require('bundle-phobia-cli').fetchPackageStats.fetchPackageStats;

const databasePath = path.join(__dirname, '../audits/byte-efficiency/bundlephobia-database.json');
```

if we're going to be mixing cwd-relative and file-relative uses of this path it's best to make sure it's absolute or it'll only work when the base is the same :)